include/envoy/stats/histogram.h

```diff
@@ -46,16 +46,17 @@ class HistogramStatistics {
   virtual const std::vector<double>& supportedBuckets() const PURE;
 
   /**
-   * Returns computed bucket values during the period. The vector contains an approximation
-   * of samples below each quantile bucket defined in supportedBuckets(). This vector is
-   * guaranteed to be the same length as supportedBuckets().
+   * Returns computed bucket values during the period. The vector contains an
+   * approximation of samples below each quantile bucket defined in
+   * supportedBuckets(). This vector is guaranteed to be the same length as
```

Comment:
can you revert this change? It looks like it is a reformatting; might as well leave it as it was.

Do you have some kind of automatic comment reformatter in your IDE or something?

Reply:
`clangd` reformatting based on `.clang-format`, will revert.

Reply:
Seems like a `clang-format` glitch or something, reverted and saved and it did not reformat again but kept as-is.